<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HqcMainWindow</class>
 <widget class="QMainWindow" name="HqcMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HQC</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0">
       <property name="spacing">
        <number>2</number>
       </property>
       <property name="margin">
        <number>2</number>
       </property>
       <item>
        <widget class="QTabWidget" name="tabs">
         <property name="tabsClosable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="saveAction"/>
    <addaction name="separator"/>
    <addaction name="actionStartKro"/>
    <addaction name="separator"/>
    <addaction name="printAction"/>
    <addaction name="actionScreenshot"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuData">
    <property name="title">
     <string>&amp;Data</string>
    </property>
    <addaction name="actionDataAndErrorList"/>
    <addaction name="actionErrorDataSalen"/>
    <addaction name="actionErrorList"/>
    <addaction name="dataListAction"/>
    <addaction name="actionErrorListSalen"/>
    <addaction name="separator"/>
    <addaction name="actionWatchRR"/>
    <addaction name="actionWatchWeather"/>
    <addaction name="separator"/>
    <addaction name="actionTextDataList"/>
    <addaction name="actionRejectDecode"/>
    <addaction name="separator"/>
    <addaction name="actionJumpToObservation"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionHelpManual"/>
    <addaction name="actionHelpNews"/>
    <addaction name="actionHelpFlags"/>
    <addaction name="actionHelpParameters"/>
    <addaction name="separator"/>
    <addaction name="actionAboutHqc"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionSettings"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuData"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockHistory">
   <property name="minimumSize">
    <size>
     <width>256</width>
     <height>357</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Change History</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="spacing">
      <number>2</number>
     </property>
     <property name="margin">
      <number>2</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QToolButton" name="toolButtonUndo">
         <property name="text">
          <string>Undo</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButtonRedo">
         <property name="text">
          <string>Redo</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButtonSave">
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTreeView" name="treeChanges"/>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="saveAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="printAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
   <property name="toolTip">
    <string>Print</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionDataAndErrorList">
   <property name="text">
    <string>Data &amp;list and Error list</string>
   </property>
   <property name="toolTip">
    <string>Produce datalist and error list</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionErrorList">
   <property name="text">
    <string>&amp;Error list</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="dataListAction">
   <property name="text">
    <string>&amp;Data list</string>
   </property>
  </action>
  <action name="actionErrorListSalen">
   <property name="text">
    <string>&amp;Error list salen</string>
   </property>
   <property name="shortcut">
    <string>Alt+S</string>
   </property>
  </action>
  <action name="actionWatchRR">
   <property name="text">
    <string>&amp;Precipitation</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionWatchWeather">
   <property name="text">
    <string>&amp;Weather</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionTextDataList">
   <property name="text">
    <string>Te&amp;xt data</string>
   </property>
   <property name="shortcut">
    <string>Alt+X</string>
   </property>
  </action>
  <action name="actionRejectDecode">
   <property name="text">
    <string>Re&amp;jected</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+J</string>
   </property>
  </action>
  <action name="actionDianaConfig">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Diana presentation</string>
   </property>
  </action>
  <action name="actionStartKro">
   <property name="text">
    <string>&amp;Open Kro…</string>
   </property>
   <property name="toolTip">
    <string>Open kro in browser</string>
   </property>
  </action>
  <action name="actionHelpManual">
   <property name="text">
    <string>&amp;User manual…</string>
   </property>
   <property name="iconText">
    <string>User manual…</string>
   </property>
   <property name="toolTip">
    <string>Show user manual in browser</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionHelpFlags">
   <property name="text">
    <string>&amp;Flags…</string>
   </property>
   <property name="toolTip">
    <string>Show flag explanations in browser</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="actionHelpParameters">
   <property name="text">
    <string>&amp;Parameters…</string>
   </property>
   <property name="toolTip">
    <string>Show parameter explanation in browser</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
  <action name="actionAboutHqc">
   <property name="text">
    <string>&amp;About Hqc</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutRole</enum>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>About &amp;Qt</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutQtRole</enum>
   </property>
  </action>
  <action name="actionScreenshot">
   <property name="text">
    <string>&amp;Screenshot</string>
   </property>
  </action>
  <action name="actionHelpNews">
   <property name="text">
    <string>&amp;News…</string>
   </property>
  </action>
  <action name="actionErrorDataSalen">
   <property name="text">
    <string>Error+data list Salen</string>
   </property>
   <property name="iconText">
    <string>Error+data list Salen</string>
   </property>
   <property name="toolTip">
    <string>Build error list and data list for Salen</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+L</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Undo the last change. &lt;span style=&quot; font-weight:600;&quot;&gt;WARNING:&lt;/span&gt; This may be many changes at once if you edited in WatchRR or WatchWeather or accepted / rejected a time series.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="toolTip">
    <string>Redo a change.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
  <action name="actionJumpToObservation">
   <property name="text">
    <string>Jump to Observation…</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings…</string>
   </property>
   <property name="toolTip">
    <string>Change user settings</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionDataAndErrorList</sender>
   <signal>triggered()</signal>
   <receiver>HqcMainWindow</receiver>
   <slot>allListMenu()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dataListAction</sender>
   <signal>triggered()</signal>
   <receiver>HqcMainWindow</receiver>
   <slot>dataListMenu()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionErrorList</sender>
   <signal>triggered()</signal>
   <receiver>HqcMainWindow</receiver>
   <slot>errListMenu()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionErrorListSalen</sender>
   <signal>triggered()</signal>
   <receiver>HqcMainWindow</receiver>
   <slot>errLisaMenu()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionWatchRR</sender>
   <signal>triggered()</signal>
   <receiver>HqcMainWindow</receiver>
   <slot>showWatchRR()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionWatchWeather</sender>
   <signal>triggered()</signal>
   <receiver>HqcMainWindow</receiver>
   <slot>showWeather()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionStartKro</sender>
   <signal>triggered()</signal>
   <receiver>HqcMainWindow</receiver>
   <slot>startKro()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionHelpManual</sender>
   <signal>triggered()</signal>
   <receiver>HqcMainWindow</receiver>
   <slot>helpUse()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionHelpFlags</sender>
   <signal>triggered()</signal>
   <receiver>HqcMainWindow</receiver>
   <slot>helpFlag()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionHelpParameters</sender>
   <signal>triggered()</signal>
   <receiver>HqcMainWindow</receiver>
   <slot>helpParam()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAboutHqc</sender>
   <signal>triggered()</signal>
   <receiver>HqcMainWindow</receiver>
   <slot>about()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAboutQt</sender>
   <signal>triggered()</signal>
   <receiver>HqcMainWindow</receiver>
   <slot>aboutQt()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionScreenshot</sender>
   <signal>triggered()</signal>
   <receiver>HqcMainWindow</receiver>
   <slot>screenshot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionHelpNews</sender>
   <signal>triggered()</signal>
   <receiver>HqcMainWindow</receiver>
   <slot>helpNews()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionErrorDataSalen</sender>
   <signal>triggered()</signal>
   <receiver>HqcMainWindow</receiver>
   <slot>allListSalenMenu()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveAction</sender>
   <signal>triggered()</signal>
   <receiver>HqcMainWindow</receiver>
   <slot>onSaveChanges()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionUndo</sender>
   <signal>triggered()</signal>
   <receiver>HqcMainWindow</receiver>
   <slot>onUndoChanges()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionRedo</sender>
   <signal>triggered()</signal>
   <receiver>HqcMainWindow</receiver>
   <slot>onRedoChanges()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionJumpToObservation</sender>
   <signal>triggered()</signal>
   <receiver>HqcMainWindow</receiver>
   <slot>onJumpToObservation()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tabs</sender>
   <signal>tabCloseRequested(int)</signal>
   <receiver>HqcMainWindow</receiver>
   <slot>onTabCloseRequested(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>333</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>541</x>
     <y>93</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSettings</sender>
   <signal>triggered()</signal>
   <receiver>HqcMainWindow</receiver>
   <slot>onUserSettings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>showFlags()</slot>
  <slot>showOrigs()</slot>
  <slot>showMod()</slot>
  <slot>showStat()</slot>
  <slot>showHeight()</slot>
  <slot>showPos()</slot>
  <slot>onUserSettings()</slot>
  <slot>allListMenu()</slot>
  <slot>errListMenu()</slot>
  <slot>dataListMenu()</slot>
  <slot>errLisaMenu()</slot>
  <slot>showWatchRR()</slot>
  <slot>showWeather()</slot>
  <slot>timeseriesMenu()</slot>
  <slot>textDataMenu()</slot>
  <slot>rejectedMenu()</slot>
  <slot>climateStatistics()</slot>
  <slot>priority()</slot>
  <slot>temperature()</slot>
  <slot>precipitation()</slot>
  <slot>airPress()</slot>
  <slot>visuals()</slot>
  <slot>sea()</slot>
  <slot>synop()</slot>
  <slot>wind()</slot>
  <slot>plu()</slot>
  <slot>all()</slot>
  <slot>clk()</slot>
  <slot>rejectTimeseries()</slot>
  <slot>acceptTimeseries()</slot>
  <slot>dsh()</slot>
  <slot>startKro()</slot>
  <slot>screenshot()</slot>
  <slot>helpUse()</slot>
  <slot>helpFlag()</slot>
  <slot>helpParam()</slot>
  <slot>about()</slot>
  <slot>aboutQt()</slot>
  <slot>helpNews()</slot>
  <slot>allListSalenMenu()</slot>
  <slot>onUndoChanges()</slot>
  <slot>onRedoChanges()</slot>
  <slot>onSaveChanges()</slot>
  <slot>onShowChanges()</slot>
  <slot>onShowSimpleCorrections()</slot>
  <slot>onTabCloseRequested(int)</slot>
 </slots>
</ui>
